capstone-frontend/src/components/ClubPage.jsx

```diff
@@ -125,6 +125,25 @@ class ClubPage extends Component {
       });
   }
 
+  handleMemberChange = (member, type) => {
+    if (type === 'add') {
+
+      const club = this.state.club;
+      club.memberIDs.push(member.id);
+      
+      this.setState({ members: [...this.state.members, member], club });
+    }
+    else if (type === 'remove') {
+
+      const members = this.state.members.filter(knownMember => knownMember.id !== member.id);
```

Comment:
nit: Can you update this line to be within the 80 character limit?

Reply:
Ok